app/Models/Signup.php

```diff
@@ -3,9 +3,18 @@
 namespace Rogue\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class Signup extends Model
 {
+    use SoftDeletes;
```

Comment:
Let's add a newline after this so it's not touching the following docblock. 👾

Reply:
added!